src/constants.ts

```diff
@@ -20,3 +20,5 @@ export enum Events {
 }
 
 export const CacheFilename = "cache.tgz";
+
+export const SocketTimeout = 5000;
```

Comment:
How did we decide on this value? Are there any recommendations or guidelines from Azure Blog Storage?

Reply:
I don't think there are any guidelines.  This value has worked well in practice in the [cache-canary](https://github.com/dhadka/cache-canary/actions) repo, as I haven't seen it abort any downloads that were still working...it's only aborted downloads where traffic over the socket stopped.